### DISCUSSION TITLE
Syntax definition

### DISCUSSION THREAD

**Discussion by nicholas-ochoa** [moudey/Shell#82] [General]
Is there a syntax highlighter definition for use in any editors? Ideally Sublime Text, or even VS Code. Is the configuration syntax based on any existing language?

I've been able to use a "generic config" syntax in sublime to add some basic highlighting but it's definitely not perfect.

**Comment by moudey** [comment]
The structure of config file and expressions is very simple and somewhat similar to the languages of C/C++, C#, Java, Javascript.

**Comment by pa-0** [comment]
> Is there a syntax highlighter definition for use in any editors? Ideally Sublime Text, or even VS Code. Is the configuration syntax based on any existing language?
> 
> I've been able to use a "generic config" syntax in sublime to add some basic highlighting but it's definitely not perfect.

FYI there are now syntax definitions available for vs-code (marketplace) and notepad-plus-plus (github)
